bedrock/settings/base.py

```diff
@@ -1607,10 +1609,14 @@ def before_send(event, hint):
     "FI",  # Finland
 ]
 
+VPN_AFFILIATE_COUNTRIES = ["CA", "DE", "FR", "IE", "UK", "US"]
 VPN_AVAILABLE_COUNTRIES = 17
 VPN_CONNECT_SERVERS = 400
 VPN_CONNECT_COUNTRIES = 30
 VPN_CONNECT_DEVICES = 5
 
 # VPN client ID for referral parameter tracking (issue 10811)
 VPN_CLIENT_ID = "e6eb0d1e856335fc"
+
+# VNP affiliate micro service (CJMS) endpoint (issue 11212)
+VPN_AFFILIATE_ENDPOINT = "https://stage.cjms.nonprod.cloudops.mozgcp.net/aic" if DEV else "https://cjms.mozilla.org/aic"  # Prod URL TBD
```

Comment:
Note for reviewer: https://cjms.mozilla.org is not live yet (and domain choice may change), but this is all behind a feature switch whilst QA happens on www-dev. Plan is to update the production end-point once it's ready.